src/main/java/uk/gov/hmcts/darts/audio/mapper/GetAdminMediaResponseMapper.java

```diff
@@ -22,18 +22,18 @@
 
 @Slf4j
 @UtilityClass
-public class AdminMediaSearchResponseMapper {
+public class GetAdminMediaResponseMapper {
```

Comment:
AdminGetMediaResponseMapper?

Reply:
As discussed, i wanted to keep 'Admin' and 'Media' together, as thats whats in the endpoint url:-
GET /admin/media

OpenapiGenterator will generate it like this:-
AdminMediaGet

so i can change it to that if it makes more sense